var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -37,5 +39,37 @@ class Coreutils(AutotoolsPackage):
     version('8.29', '960cfe75a42c9907c71439f8eb436303')
     version('8.26', 'd5aa2072f662d4118b9f4c63b94601a6')
     version('8.23', 'abed135279f87ad6762ce57ff6d89c41')
+    version('8.22', '8fb0ae2267aa6e728958adc38f8163a2')
 
     build_directory = 'spack-build'
+
+    variant('mutil', default=False, description='Enable - Multi-threaded cp and md5sum based on GNU coreutils')
```

Comment:
I would use the description "Multi-threaded cp and md5sum"

"Enable" is not accurate if you're doing `~mutil` and we already know it is GNU coreutils.